.github/ISSUE_TEMPLATE/bugs-performance-issues.md

```diff
@@ -2,6 +2,7 @@
 name: Bug report (including performance and build issues)
 about: Help us find mistakes in the GPflow code base
 labels: bug
+projects: 7
```

Comment:
What does this "projects: 7" do?

Reply:
I'm not sure if it'll work (the github docs were inconclusive), but *hopefully* it'll automatically add them to [project board number 7](https://github.com/GPflow/GPflow/projects/7), which is our Open bugs tracker